adapters/repos/db/inverted/objects.go

```diff
@@ -473,3 +507,43 @@ func (a *Analyzer) analyzeRefProp(prop *models.Property,
 		HasFrequency: false,
 	}, nil
 }
+
+func typedSliceToUntyped(in interface{}) ([]interface{}, error) {
+	switch typed := in.(type) {
+	case []interface{}:
+		// nothing to do
+		return typed, nil
+	case []string:
+		out := make([]interface{}, len(typed))
+		for i := range out {
+			out[i] = typed[i]
+		}
+		return out, nil
+	case []int:
+		out := make([]interface{}, len(typed))
+		for i := range out {
+			out[i] = typed[i]
+		}
```

Comment:
test?

Reply:
The fact this is uncovered makes me wonder if we can ever land in this code path. I'll try to find out more and either remove it or add an explicit test for it. My gut feeling is that this may be required for auto-schema... (maybe @antas-marcin knows more?)

Reply:
This is covered now